src/sub-blocks/Quote/Quote.scss

```diff
@@ -7,7 +7,6 @@ $block: '.#{$ns}quote';
     &_#{$name} {
         &::before {
             position: absolute;
```

Comment:
seems like for other quote type it is still actual

Reply:
here it already applied for the left quote only, but for all `quoteType`'s. No need to align for the right one